### PR TITLE
Add column visibility to jobs table

### PR DESCRIPTION
**Pull request by rafiramadhana**
Changes:

- Add migration files
- Update sqlc and related tests
  - Add `JobHide` to update `jobs.visibility` to `"hidden"`

Issue: https://github.com/optriment/optrispace-backend/issues/139

### CODE REVIEW

File: pkg/db/sqlc/queries/jobs.sql [hunk 1]
```diff
@@ -11,9 +11,10 @@ select
     ,(select count(*) from applications a where a.job_id = j.id) as application_count
     ,(CASE WHEN p.display_name = '' THEN p.login ELSE p.display_name END)::varchar AS customer_display_name
     ,p.ethereum_address AS customer_ethereum_address
+    ,j.visibility
```
Comment (gruz0) [inline pkg/db/sqlc/queries/jobs.sql]: I checked it, maybe it would be good to remove this column from this query, because we will not use it on frontend.

Comment (gruz0) [reply pkg/db/sqlc/queries/jobs.sql]: And after this please execute `make sqlc` in the project root folder to update sqlc models 🙏

**Comment by gruz0** [comment]
LGTM

File: pkg/db/sqlc/queries/jobs.sql (hunk 1 above)
Comment (rafiramadhana) [reply pkg/db/sqlc/queries/jobs.sql]: ok will do the change

Comment (rafiramadhana) [reply pkg/db/sqlc/queries/jobs.sql]: updated

https://github.com/optriment/optrispace-backend/pull/143/commits/fc8914b164d646e258872cddcb6b0e736fbcf90b

**Comment by rafiramadhana** [comment]
@gruz0  if all is well, you can have this merged

im doing this for free and more into learning opportunity

thank you :rocket:

**Comment by gruz0** [comment]
Thank you very much for your help, @neverbeenthisweeb!

BTW if you want to get paid for this task in crypto (BNB), simply apply on this task on our platform: 
https://my.optrispace.com/jobs/ULKtiVBowV7aBfQkwXaE7g/

**Pull request merged**